expenses/reports.py

```diff
@@ -90,7 +123,12 @@ class Report(metaclass=abc.ABCMeta):
 
     @classmethod
     def meta_to_dict(cls) -> typing.Dict[str, typing.Any]:
-        return {"name": cls.name, "slug": cls.slug, "description": cls.description, "options": cls.options}
+        return {
```

Comment:
It looks like you ran black with line-length=88; the standard for this project is 120 and `pyproject.toml` should enforce that — please reformat with the correct length.

Reply:
(I didn’t really check the rest of the code, because it would require a lot of effort to check if a diff is an actual change or a formatting change.)

Reply:
I'm sorry for the issue with black. I accidentally changed my editor's settings and it resulted in wrong formatting